src/interfaces/coral_web/src/hooks/chat.ts

```diff
@@ -248,28 +292,31 @@ export const useChat = (config?: { onSend?: (msg: string) => void }) => {
 
               saveCitations(generationId, citations, documentsMap);
 
+              const finalText = isRAGOn
+                ? replaceTextWithCitations(
+                    // TODO(jessica): temporarily use the text generated from the stream when MAX_TOKENS
```

Comment:
feel like we should use handles instead of names for this repo 

```suggestion
                    // TODO(@wujessica): temporarily use the text generated from the stream when MAX_TOKENS
```

Reply:
good call!